bot/moderation/moderation.py

```diff
@@ -82,6 +82,7 @@ async def lockdown(self, ctx: commands.Context,
             overwrite.update(send_messages=False, connect=False)
             await channel.set_permissions(ctx.guild.default_role, overwrite=overwrite,
                                           reason=f"Der Kanal wurde von {ctx.author} in einen Lockdown versetzt.")
+            log.info("Der Kanal [#%s] wurde in einen Lockdown versetzt.", channel)
```

Comment:
Waren unsre log messages nicht englisch?
Das sollte nicht gemischt werden.

Reply:
No kruzifix noimol. Des is komplett richtig!